src/common/config.go

```diff
@@ -131,7 +134,7 @@ type LimitsConfig struct {
 
 // Choose reasonable defaults for a worker deployment (based on memory capacity).
 // olPath need not exist (it is used to determine default paths for registry, etc).
-func LoadDefaults(olPath string) error {
+func LoadDefaults(olPath string, dockerImg string) error {
```

Comment:
It feels a bit off that we tell a function called LoadDefaults the specific value for one of the fields.  I think it is better for LoadDefaults to not take dockerImg and hardcode a default for Docker_base_image.  A caller of LoadDefaults can override then override that field in the returned struct if it wishes.